docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -17,7 +17,53 @@ Quarkus extensions add a new developer focused behavior to the core offering, an
 This means that metadata is only processed once at build time, which both saves on startup time, and also on memory
 usage as the classes etc that are used for processing are not loaded (or even present) in the runtime JVM.
 
-== Three Phases of Bootstrap and Quarkus Philosophy
+== Extension philosophy
+
+This section is a work in progress and gathers the philosophy under which extensions should be designed and written.
+
+=== Why an extension framework
+
+Quarkus’ mission is to transform your full applications and the libraries it uses into an optimal output for GraalVM.
```

Comment:
Quarkus's mission (yeah, English is weird, I know) ... your full application and the libraries

Reply:
@LightGuard funny enough @n1hility wrote that part. I guess Madison English is different than US English :D